modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/adapters/BundleGroupDocAdapter.java

```diff
@@ -136,9 +137,23 @@ public String getArtifactType() {
         return TYPE_NAME;
     }
 
+    @Override
+    public BundleGroup getParentGroup() {
+        return getCoreSession().getParentDocument(doc.getRef()).getAdapter(BundleGroup.class);
+    }
+
     @Override
     public List<String> getParentIds() {
-        throw new UnsupportedOperationException();
+        List<DocumentModel> parents = getCoreSession().getParentDocuments(doc.getRef());
+        Collections.reverse(parents);
+        List<String> res = new ArrayList<>();
+        for (DocumentModel doc : parents) {
+            BundleGroup bgroup = doc.getAdapter(BundleGroup.class);
+            if (bgroup != null) {
+                res.add(bgroup.getId());
+            }
+        }
+        return res;
```

Comment:
Could be

        return parents.stream()
                      .map(doc -> doc.getAdapter(BundleGroup.class))
                      .filter(Objects::nonNull)
                      .map(BundleGroup::getId)
                      .collect(Collectors.toList());